react/ui-components/src/molecules/ApiDropdown.js

```
@@ -12,22 +12,12 @@ const ApiDropdown = ({ populators, formData, props, inputRef, errors ,disabled})
   const reqCriteria = Digit?.Customizations?.[populators?.masterName]?.[populators?.moduleName]?.[populators?.customfn]();
-  const {
-    isLoading: isApiLoading,
-    data: apiData,
-    revalidate,
-    isFetching: isApiFetching,
-  } = reqCriteria
-    ? window?.Digit?.Hooks.useCustomAPIHook(reqCriteria)
-    : { isLoading: false, data: [], revalidate: null, isFetching: false };
 
-  const memoizedApiData = useMemo(() => apiData, [JSON.stringify(apiData)]);
+  const { isLoading: isApiLoading, data: apiData, revalidate, isFetching: isApiFetching } = window?.Digit?.Hooks.useCustomAPIHook(reqCriteria);
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)